website/blog/2016-07-27-jest-14.md

```diff
@@ -63,7 +63,11 @@ You can start using Jest with react-native by running `yarn add --dev jest-react
 - [Example project](https://github.com/facebook/jest/tree/main/examples/react-native)
 - [Example pull request for _snowflake_](https://github.com/bartonhammond/snowflake/pull/110), a popular react-native open source library.
 
-_Note: the preset currently only implements the minimal set of configuration necessary to get started with React Native testing. We are hoping for community contributions to improve this project. Please try it and file [issues](https://github.com/facebook/jest/issues) or send pull requests!_
+:::note
```

Comment:
Also it would be nice to split blog posts into separate PR. All of the blog posts in one chuck.